### ISSUE TITLE
No way to go from a patterns page to the authors profiles.wordpress.org page

### ISSUE THREAD

**Issue by benniledl** [WordPress/pattern-directory#696]
Hey,

In the old design, it was possible to navigate from a pattern page to the author's WordPress.org profile by clicking on the profile picture. Clicking on the author's username would take you to the author's page where you could see all patterns submitted by that author.

Expected Behavior:

1. When viewing a pattern page https://wordpress.org/patterns/pattern/cover/
2. Clicking on the author's username should take you to: https://wordpress.org/patterns/author/benniledl/
3. Clicking on the profile picture should take you to: https://profiles.wordpress.org/benniledl/

Actual Behavior:

1. When viewing a pattern page https://wordpress.org/patterns/pattern/cover/
2. Clicking on either the username or the profile picture both take you to: https://wordpress.org/patterns/author/benniledl/

**Comment by ryelle** [comment]
> In the old design, it was possible to navigate from a pattern page to the author's WordPress.org profile by clicking on the profile picture.

I don't think it did, the username and image were [both in the same link](https://github.com/WordPress/pattern-directory/blob/trunk/public_html/wp-content/themes/pattern-directory/src/components/pattern/author-details.js). But I do like the suggestion to have a way to get to the profile.

Maybe from the author archive? On the Theme Directory, author archive pages have a heading ([example](https://wordpress.org/themes/author/wordpressdotorg/)), we could make the username there a link to the w.org profile and use the same pattern on Patterns?

![Screen Shot 2024-06-20 at 10 12 08](https://github.com/WordPress/pattern-directory/assets/541093/1c931e85-246c-4053-a742-7d7a94bc945b)

**Comment by benniledl** [comment]
Hey, oops maybe I mixed things up, I know this behavior as I described is present in trac, but I thought the pattern directory worked the same in the old design.

Your suggestions seems nice :)

**Comment by jasmussen** [comment]
There may be a better design for navigating to the author profile. That's worth considering as part of the log-in/log-out/register part of the flow that's already being discussed in a few places (i.e. maybe a little circular avatar can appear next to wherever we end up adding those sign-in links), but that can be a longer term exploration.

If Kelly's design solves the problem in the meantime, seems okay to ask. I might make the author link black, same as the heading title, instead of blue, though, that matches [the Documentation page](https://wordpress.org/documentation/).
